### PR TITLE
Grace updates 6-8

### PR DESCRIPTION
**Pull request by gracecanfield**
Addresses the following Workast tasks:
* [More wiring info](https://respiraworks.workast.io/task/2e3363f0effb486fe3f6392e2d015fff)
  * Note: It looks like 2 of the 3 parts listed were already in the BOM, so only added 22-gauge wiring as a new addition.
* [Synchronize simple and compliance test lung page](https://respiraworks.workast.io/task/933024309b3f73528edd4f5f9a9df3c5)
  * Note: I wasn't sure about several aspects of this task - I'll leave questions in a separate comment.

### CODE REVIEW

**Comment by gracecanfield** [comment]
@martukas For the [Workast task to synchronize simple and compliance test lung pages](https://respiraworks.workast.io/task/933024309b3f73528edd4f5f9a9df3c5), I had a few questions:
1. The subtasks noted "remove redundancies in BOMs". I took a stab at this, but I'm not sure if I got it right:
    * I didn't see any literal part number redundancies between the BOMs, but I did see a couple different parts that seemed redundant between the two. I took a stab at describing these redundancies [here](https://github.com/RespiraWorks/SystemDesign/tree/grace-updates-6-8/ventilator-build/alpha-build-instructions/pizza-compliance-test-lung#part-list-bom), but I'm sure you might have tweaks or other thoughts.
    * It looks like the test lung itself is replaced in the reduced-compliance test lung with a (more expensive) version, so I'm currently telling people to not buy the version in the simple test lung page if they're planning to build a reduced-compliance test lung. Does this feel right?
    * It looks like we're giving people several part recommendations for an "Alt Tee" in the [simple test lung page](https://github.com/RespiraWorks/SystemDesign/tree/grace-updates-6-8/ventilator-build/alpha-build-instructions/pizza-test-lung) (all of the `a` items), but then we have a single "tee" part listed (item 6) in the [reduced compliance lung page](https://github.com/RespiraWorks/SystemDesign/tree/grace-updates-6-8/ventilator-build/alpha-build-instructions/pizza-compliance-test-lung#part-list-bom). Is it correct that someone building the reduced compliance test lung can disregard the entire "Alt Tee" table in the simple test lung page?
    * Are there any other redundancies I missed?
2. There was a subtask to "Get new pictures". Do you have any new pictures you'd like to add or replace?

**Comment by gracecanfield** [comment]
@martukas Also just pushed some additional commits for the Workast task to [update all pages with purchasing info to include instructions[(https://respiraworks.workast.io/task/4a0e40c9f5ee950c9afaf011cd0e166c), but had one question for you: I included a mention [on the ventilator-build page](https://github.com/RespiraWorks/SystemDesign/tree/grace-updates-6-8/ventilator-build) asking team members in Slack before purchasing any parts. Is `#beta-build` the right place to direct people to, or is there a better channel for parts-related communication?

**Comment by gracecanfield** [comment]
@martukas Also - can you think of any reason we'd need to keep the "Action" column on any of the BOMs, now that we'll have a universal mention of asking team members before purchasing on the ventilator-build page? Wanted to confirm this with you before deleting that column on all of the BOMs.

**Comment by gracecanfield** [comment]
Final update: Just committed changes to pizza & brain build pages to [add missing Manufacturer & Part numbers](https://respiraworks.workast.io/task/02d44e495808d6df0889d9fe54b54beb).

File: ventilator-build/README.md [hunk 1]
```diff
@@ -4,3 +4,16 @@ This contains details on the alpha and beta build assemblies, including parts in
 
 * [Alpha Build Instructions](alpha-build-instructions)
 * [Beta Build Plan](beta-build-instructions)
+
+# Part Purchasing Guidelines
+
+If you are a member of the RespiraWorks team, please don’t buy anything using personal funds (we need to document everything in specific ways to meet the requirements of our grant funding).
+
+Before purchasing any parts, **ask** team members (the [#beta-build](https://respiraworks.slack.com/archives/C0122FX6G90) RespiraWorks Slack channel is a good place to start) if they have any spare parts floating around. There are a lot of extras, so you may be able to get your hands on parts without ordering anything.
```
Comment (martukas) [inline ventilator-build/README.md:12]: I think it should be #dev-hardware instead

File: ventilator-build/README.md [hunk 2]
```diff
@@ -4,3 +4,16 @@ This contains details on the alpha and beta build assemblies, including parts in
...
+
+Before purchasing any parts, **ask** team members (the [#beta-build](https://respiraworks.slack.com/archives/C0122FX6G90) RespiraWorks Slack channel is a good place to start) if they have any spare parts floating around. There are a lot of extras, so you may be able to get your hands on parts without ordering anything. 
+
+If you do need to purchase parts, follow the process below. Note: This process is subject to change as we work out any issues. Message Curtis Kline with any questions.
+
+1. If you need to buy something, put the details into a post in [#purchasing-sourcing](https://respiraworks.slack.com/archives/C011FD2TEQM) (RespiraWorks Slack channel) and tag the [@purchasing group](https://respiraworks.slack.com/admin/user_groups).
+2. One of the Purchasing team members will source the items and confirm with you before placing the order (unless it’s obvious).
+3. Everything will be purchased using a RespiraWorks credit card whenever possible.
+4. The Purchasing team member will reply to your message in [#purchasing-sourcing](https://respiraworks.slack.com/archives/C011FD2TEQM) with tracking info when available.
```
Comment (martukas) [inline ventilator-build/README.md:19]: I would add 5. saying that:
if you **really really** need something ASAP, we can reimburse you. But please check in with the leads first and don't abuse this.
Or do you think best leave this unsaid? :)

File: ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md [hunk 3]
```diff
@@ -1,19 +1,25 @@
...
 ![Completed](assets/test-lung-4.jpg)
 
 To test basic function a ventilator check lung or a breathing bag is sufficient, but can really only work up to 10cmH2O.
-To be able to test higher PIP values, you need to reduce the conpliance of the lung so you don't just inflate it like
-a balloon (or pop it).  These are partial instructions, and there is currently a shortage of test lungs, so what model
-you get can vary, so expect to do some tinkering to these instrutions if you are going to build you own.
-This is completely uncalibrated, so it is not intended for instrumentation just for functional observation.
 
-**#TODO: synchronize parts list in this page with that in [simple test lung](../pizza-test-lung/README.md), or make that a prerequisite**
+To be able to test higher PIP values, you need to reduce the conpliance of the lung so you don't just inflate it like a balloon (or pop it).  These are partial instructions, and there is currently a shortage of test lungs, so what model you get can vary, so expect to do some tinkering to these instrutions if you are going to build you own.
```
Comment (martukas) [inline ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md:9]: No, please don't concatenate everything into one line. I realize that you have your word-wrap turned on in your IDE and it makes things look "cleaner" when editing. But MD doesn't care about single carriage returns and puts everything in one paragraph anyways. It's much easier for diffing and change tracking to have more lines not fewer.

File: ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md [hunk 4]
```diff
@@ -1,19 +1,25 @@
...
+This is completely uncalibrated, so it is not intended for instrumentation, just for functional observation.
+
+**NOTE: As a prerequisite for this reduced-compliance test lung, please first follow instructions and buy parts as needed for the [simple test lung](../pizza-test-lung). See additional notes on part redundancies in the Part List section below.**
 
 ## Part List (BOM)
 
+**Note: If you are a member of the RespiraWorks team, review [part purchasing guidelines](../../README.md) BEFORE purchasing any parts.**
+
+**Add-on part redundancies: You'll first need to secure parts for the [simple test lung](../pizza-test-lung), EXCEPT for the following:**
+*  **DON'T purchase Item #1 ion the simple test lung BOM; instead purchase item #1 in the BOM below.**
```
Comment (martukas) [inline ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md:20]: No, there is no preference for this. This is just an alternative that can be moved into the regular test lung page.

File: ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md [hunk 5]
```diff
@@ -1,19 +1,25 @@
...
+* **DON'T purchase parts within the "Parts: Alt Tee" table in the simple test lung BOM; instead, purchase item #6 in the BOM below.**
```
Comment (martukas) [inline ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md:21]: Same here. This is one the redundancies. Has no business on this page. How you build the lung and connect it pneumatically has nothing to do with how you control the compliance.

**Comment by martukas** [comment]
I think I answered your questions in specific spots. No, there is no need for the action column anymore if we have those general purchasing instructions.

File: ventilator-build/README.md (hunk 1 above)
Comment (gracecanfield) [reply ventilator-build/README.md]: Updated.

File: ventilator-build/README.md (hunk 2 above)
Comment (gracecanfield) [reply ventilator-build/README.md]: It seems like if it was that urgent they would talk to someone, and that this is probably enough of an edge case that we don't have to include it in written instructions. But, I also don't feel super-strongly about this.

File: ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md (hunk 3 above)
Comment (gracecanfield) [reply ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md:9]: Hmm...I didn't concatinate everything into one line. Everything was bundled into a single paragraph before, with the exception of a couple rando single-line breaks in the middle of sentences. So, I separated each line into its own paragraph (to be more easily editable and readable). The only line breaks I eliminated were the 2 arbitrary ones in the middle of the sentence starting with "To be able to test higher PIP values,..."

File: ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md (hunk 5 above)
Comment (gracecanfield) [reply ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md:21]: >This is the one the redundancies.

Are you referring to item 6? (And if so, are we saying remove item 6 entirely, OR list itbut on the simple test lung page, in the main Parts table, with the "alt tee" remaining as an...alt?)

File: ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md (hunk 4 above)
Comment (gracecanfield) [reply ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md:20]: Removed from this page; moved as an alt into the simple test lung page instead.

**Comment by gracecanfield** [comment]
> I think I answered your questions in specific spots. No, there is no need for the action column anymore if we have those general purchasing instructions.

Cool - that's in a separate workast branch but i'll update it soon.

File: ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md (hunk 5 above)
Comment (martukas) [reply ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md:21]: Just move it as another alt tee, saying something like, depends on your tubing, use your own judgement, etc. The problem is usually not that you can't tell if you'll want it, the problem is that you have no idea what you could even pick out of the millions of head-spinning varieties of tubes that McMaster sells.

File: ventilator-build/README.md (hunk 2 above)
Comment (martukas) [reply ventilator-build/README.md]: Ok, well at least we could say, that in the rare case that you do buy something essential, send your receipts to #accounting-reimburse

Comment (gracecanfield) [reply ventilator-build/README.md]: Okay, well that is basically saying the same thing except with more specific instruction. Next time, just say if you have a strong preference - it'll save time. I'll update.

File: ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md (hunk 5 above)
Comment (gracecanfield) [reply ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md]: Done.

File: ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md (hunk 3 above)
Comment (gracecanfield) [reply ventilator-build/alpha-build-instructions/pizza-compliance-test-lung/README.md]: Updated
